sdk/mediaservices/Azure.ResourceManager.Media/api/Azure.ResourceManager.Media.netstandard2.0.cs

```diff
@@ -2292,6 +2596,26 @@ public partial class MediaVideoBase : Azure.ResourceManager.Media.Models.MediaCo
         public Azure.ResourceManager.Media.Models.InputVideoStretchMode? StretchMode { get { throw null; } set { } }
         public Azure.ResourceManager.Media.Models.VideoSyncMode? SyncMode { get { throw null; } set { } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct MinimumTlsVersion : System.IEquatable<Azure.ResourceManager.Media.Models.MinimumTlsVersion>
```

Comment:
Please add a `Media` prefix to the model name and use custom code to rename the value to a format like `Tls1_0`. You can refer to [this file](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.ResourceManager.ServiceBus/src/Customized/ServiceBusMinimumTlsVersion.cs).